docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3.9'
+services:
+  postgres:
+    container_name: op_ent_db
+    image: postgres:alpine
+    volumes:
+      - op_ent_db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: 'op_ent'
+      POSTGRES_USER: 'op_ent'
+      POSTGRES_PASSWORD: 'Mic0rk5en!sCool'
+    ports:
+      - 5432:5432
+
+  redis:
+    container_name: op_ent_redis
+    image: redis:alpine
+    volumes:
+      - op_ent_redis:/data
+    ports:
+      - 6379:6379
+
+  adonis:
+    container_name: op_ent_web
+    restart: always
+    depends_on:
+      - postgres
+      - redis
+    image: 'lucasrgoes/adonis'
+    ports:
+      - 3333:3333
+    env_file:
+      - .env
+    environment:
+      - PG_HOST=postgres
+      - REDIS_HOST=redis
+    volumes:
+      - type: bind
+        source: ./
+        target: /app
+    command: dumb-init node ace serve --watch

```

Comment:
update to `dumb-init node ace serve --watch --node-args="--inspect=0.0.0.0"`

Reply:
Why is there a need to inspect?

Reply:
I'm asking on the adonis discord server

Reply:
Okay, contact me on Discord for the response.